src/createSliceSaga.ts

```diff
@@ -105,15 +106,13 @@ export function createSaga(
   }
 }
 
-export function createSagas(
-  sagas: (() => Generator<unknown, void, SagaIterator>)[],
-): any {
-  const sagaTemp = [];
-  sagas.forEach((saga: any) => {
-    sagaTemp.push(call(saga));
+export function createRootSaga(callEffects: CallEffects[]): any {
```

Comment:
Rename this function to `createSagas`. Because `createSagas` was used before.
The `callEffects` parameter must be of type` (() => Generator <unknown, void, SagaIterator>) [] `. If it's `CallEffects []`, some places break the type.

Reply:
OKKK